datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - datadog_checks_base
 
+## 7.0.0 / 2019-04-18
+
+* [Added] Add TLS integration. See [#3256](https://github.com/DataDog/integrations-core/pull/3256).
```

Comment:
Should we manually update this to reference specific deps aded?

Reply:
done